en/examples/menu.json

```diff
@@ -2,7 +2,7 @@
   {
     "title": "Essentials",
     "links": [
-      { "name": "Hello world", "to": "" },
+      { "name": "Hello World", "to": "" },
```

Comment:
The « Hello World » term is usually in capitals.
— Source: https://en.wikipedia.org/wiki/Hello_World_(disambiguation)